### DISCUSSION TITLE
Improve of simulation fit to experimental data of LFP cell

### DISCUSSION THREAD

**Discussion by ALMA7867** [pybamm-team/PyBaMM#4550] [General]
This is LFP battery comparision, I have changed only Electrode height, width, number of cells in parallel, and nominal cell capacity and current function in Prada2013 parameter set, making it C/5 C-rate and 50 Ah capacity. I just want to Know, what parameters affect the flatness of Discharge capacity vs. Voltage curve (Q-V curve). Like in this case experimental curve is more flat while simulation is little steep (inclined).
![image](https://github.com/user-attachments/assets/091bca63-52b7-459b-9b5c-6413ef60da98)

How Can I more improve the simulation result?

**Comment by valentinsulzer** [comment]
Typically you would plot voltage on the y axis vs capacity on the x axis.
The main parameters to change are OCP and particle diffusivity
